.github/workflows/early_final_release_checker.yml

```diff
@@ -0,0 +1,122 @@
+---
+name: early-final-release-checker
+
+on:
+  workflow_dispatch:
+    inputs:
+      small_runner:
+        description: 'Small runner: Comma-separated list of datasets to archive (e.g., "ferc2","ferc6").'
+        # We can't pass env variables to the workflow_dispatch, so we manually list all small datasets here.
+        default: '"eia860m","eia861","eia923"'
+        required: true
+        type: string
+      create_github_issue:
+        description: "Create a Github issue from this run?"
+        default: false
+        required: true
+        type: boolean
+  schedule:
+    - cron: "21 8 2-31 6-7,9-10 *" # 8:21 AM UTC, every day in June, July, Sept, Oct except the 1st
```

Comment:
Every day except for the 1st day of the month, when the big archiver job will be running